src/components/molecules/UiPhoneNumber/UiPhoneNumber.vue

```diff
@@ -35,18 +35,22 @@
 </template>
 
 <script setup lang="ts">
-import { computed } from 'vue';
+import {
+  computed,
+  useAttrs,
+} from 'vue';
 import { uid } from 'uid/single';
 import UiPhoneNumberPrefix from './_internal/UiPhoneNumberPrefix/UiPhoneNumberPrefix.vue';
 import UiPhoneNumberInput from './_internal/UiPhoneNumberInput/UiPhoneNumberInput.vue';
+import type { DefineAttrsProps } from '../../../types';
 import type {
   PhoneCodeType,
   SupportedCountryCodeType,
 } from './helpers';
 
 export type CountryCodes = (PhoneCodeType & Record<string, unknown>)[];
 
-export interface UiPhoneNumberProps {
+export interface PhoneNumberProps {
```

Comment:
Related to https://component.infermedica.com/?path=/docs/contributing-guide-typescript-guidelines--docs#attrs-props

> Naming
Use the component name as a prefix, followed by the word AttrsProps as a suffix for the type of the component's inherited attributes. For example, MultipleAnswerAttrsProps for the UiMultipleAnswer component.